app/Models/Smartcars/Pirep.php

```diff
@@ -59,6 +59,12 @@ public function bid()
         return $this->belongsTo(\App\Models\Smartcars\Bid::class, 'bid_id', 'id');
     }
 
+    public function account()
+    {
+        return $this->hasManyThrough(\App\Models\Mship\Account::class, \App\Models\Smartcars\Bid::class,
```

Comment:
I know you're following the trend of the model but can we import all of the classes being referenced as use statements please?